src/java/org/apache/cassandra/cql3/statements/schema/AlterViewStatement.java

```diff
@@ -54,6 +65,10 @@ public Keyspaces apply(Keyspaces schema)
 
         attrs.validate();
 
+        // Guardrails on table properties
+        Guardrails.disallowedTableProperties.ensureAllowed(attrs.updatedProperties(), state);
+        Guardrails.ignoredTableProperties.maybeIgnoreAndWarn(attrs.updatedProperties(), attrs::removeProperty, state);
```

Comment:
same comment as above; I feel we shouldn't need to do both warn/fail checks here as we could simplify as

```
// this performs both warn and failure checks
Guardrails.allowTableProperties.check(attrs.updatedProperties(), attrs::removeProperty, state);
```